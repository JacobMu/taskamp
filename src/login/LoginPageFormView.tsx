import React from "react";
import { InputGroup, Switch } from "@blueprintjs/core";
import { IconNames } from "@blueprintjs/icons";

export const LoginPageFormView: React.FC = () => (
  <>
    <InputGroup placeholder={"login.form.username"} />
    <InputGroup placeholder={"login.form.password"} leftIcon={IconNames.LOCK} />
    <Switch label={"login.form.rememberMe"} />
  </>
);
