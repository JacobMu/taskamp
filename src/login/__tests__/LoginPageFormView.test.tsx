import React from "react";
import { shallow } from "enzyme";
import {LoginPageFormView} from "../LoginPageFormView";

describe("<LoginPageContainer />", () => {
  it("renders Login page form", () => {
    const wrapper = shallow(<LoginPageFormView />);

    expect(wrapper).toMatchSnapshot();
  });
});
