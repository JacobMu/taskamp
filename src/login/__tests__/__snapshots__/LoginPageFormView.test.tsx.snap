// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LoginPageContainer /> renders Login page form 1`] = `
<Fragment>
  <Blueprint3.InputGroup
    placeholder="login.form.username"
  />
  <Blueprint3.InputGroup
    leftIcon="lock"
    placeholder="login.form.password"
  />
  <Blueprint3.Switch
    label="login.form.rememberMe"
  />
</Fragment>
`;
